state/azure/blobstorage/blobstorage.go

```diff
@@ -169,16 +179,10 @@ func getBlobStorageMetadata(metadata map[string]string) (*blobStorageMetadata, e
 		return nil, fmt.Errorf("missing or empty %s field from metadata", accountNameKey)
 	}
 
-	if val, ok := metadata[accountKeyKey]; ok && val != "" {
```

Comment:
This authentication method should still be supported.

Reply:
Yes, i will ensure this is added back

Reply:
It wasn't removed -- just moved to `authentication/azure/storage`